recipient.go

```diff
@@ -7,34 +7,38 @@ import (
 )
 
 // RecipientType is the list of allowed values for the recipient's type.
-// Allowed values are "individual", "corporation".
 type RecipientType string
 
+const (
+	RecipientTypeIndividual RecipientType = "individual"
+	RecipientTypeCorp       RecipientType = "corporation"
```

Comment:
Not a big deal because of recipient, but should we call this `RecipientTypeCorporation` just to kill all abbreviations?